t/api/02-iso.t

```diff
@@ -1121,5 +1121,26 @@ subtest 'schedule tests correctly when changing TEST to job template name' => su
     is($res->json->{count}, 1, 'only schedule the child job with the same job template name');
     my %child_job = map { $jobs->find($_)->TEST => 1 } @{$res->json->{ids}};
     is_deeply(\%child_job, {child_variant1 => 1}, "only one child job was scheduled");
+    $schema->txn_rollback;
 };
+
+subtest 'PUBLISH_HDD_1 is invalid' => sub {
+    $schema->txn_begin;
+    add_opensuse_test('parent');
+    add_opensuse_test(
+        'child',
+        START_AFTER_TEST => 'parent',
+        PUBLISH_HDD_1    => 'foo/foo@64bit.qcow2'
+    );
+    my $res = schedule_iso({%iso, _GROUP_ID => '1002'}, 200);
+    is($res->json->{count},                   1,       'only schedule the test suite named parent');
```

Comment:
just to make sure we use the right semantics: One can not "schedule a test suite" as a test suite is "abstract". Just call it "test" instead of "test suite" :) And better avoid imperative mood: Replace "only schedule …" with "only parent is scheduled"